kingpin/actors/aws/ecs.py

```diff
@@ -58,6 +58,7 @@ class ECSTaskFailedException(exceptions.RecoverableActorFailure):
                     'image': {'type': 'string'},
                     'cpu': {'type': 'number'},
                     'memory': {'type': 'number'},
+                    'memoryReservation': {'type': 'number'},
```

Comment:
I'd prefer allowing any schema, and making sure that a kingpin dry run actually validates the schema by uploading it to ECS. They will keep changing this